app/models/paper.rb

```diff
@@ -5,10 +5,12 @@ class Paper < ApplicationRecord
   has_and_belongs_to_many :authors
   has_many :lists, through: :references
   has_many :references, dependent: :destroy
+  belongs_to :publication
 
   accepts_nested_attributes_for :authors, reject_if: proc { |attributes| attributes['name'].blank? }
-  validates_associated :authors
+  accepts_nested_attributes_for :publication, reject_if: proc { |attributes| attributes['name'].blank? }
```

Comment:
yea, we can do that.

On Tue, Aug 23, 2016 at 2:55 PM, Brian Kung <notifications@github.com>
wrote:

> In app/models/paper.rb
> <https://github.com/briankung/projectcredo/pull/64#discussion_r75938547>:
>
> >
> >    accepts_nested_attributes_for :authors, reject_if: proc { |attributes| attributes['name'].blank? }
> > -  validates_associated :authors
> > +  accepts_nested_attributes_for :publication, reject_if: proc { |attributes| attributes['name'].blank? }
>
> You use proc { |attributes| attributes['name'].blank? } in two places.
> Should we think about maybe assigning the proc to a variable or just
> encapsulating it as a method?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/briankung/projectcredo/pull/64/files/c10ff7dd636d241cfd3c03b300def8df2400222c#r75938547>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACKsZOiX4qNRv4zaHBxAVV20WSuiAVd5ks5qi1AegaJpZM4JqR56>
> .
>